### DISCUSSION TITLE
Repeater fields with Mask breaks disable()?

### DISCUSSION THREAD

**Discussion by yaplej** [filamentphp/filament#6501] [Help] [answered]
Hello,

I have a repeater with several fields that have mask combined with disabled() the fields do not disable until the page is refreshed.  Removing the mask resolves this.  Just curious if anyone else has seen this?

```
Forms\Components\TextInput::make('shipping_cost')
                ->reactive()
                ->disabled(fn(Closure $get): bool => $get('../../purchase_request_state_id') != 1)
                ->mask(fn(Forms\Components\TextInput\Mask $mask) => $mask->money(prefix: '$', thousandsSeparator: ',', decimalPlaces: 2)),
``

**Comment by danharrin** [comment, marked as answer]
Repeater is currently not compatible with Masks due to a conflict between Alpine and our masking library. In v3 we will try and fix this

**Reply by yaplej** [reply]
Ah thanks for the update.  I will just use a prefix for now.

```
           Forms\Components\TextInput::make('shipping_cost')
                ->reactive()
                ->prefix('$')
```
